Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -3398,49 +3398,74 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
     return matches;
 }
 
-Vector<FloatRect> UnifiedPDFPlugin::rectsForTextMatch(const WebFoundTextRange::PDFData& data)
+Vector<WebCore::FloatRect> UnifiedPDFPlugin::rectsForTextMatchesInRect(const Vector<WebFoundTextRange::PDFData>& matches, WebCore::IntRect clipRect)
 {
-    RetainPtr selection = selectionFromWebFoundTextRangePDFData(data);
-    if (!selection)
+
+    RefPtr frame = m_frame.get();
+    if (!frame || !frame->page() || !frame->coreLocalFrame())
```

Comment:
Do we need to check `!frame->page()`? I don't see it used anywhere?

Reply:
copy pasteo (though the original is wrong too)